Content.Server/Body/Systems/InternalsSystem.cs

```diff
@@ -113,7 +113,8 @@ private short GetSeverity(InternalsComponent component)
     {
         if (component.BreathToolEntity == null || !AreInternalsWorking(component)) return 2;
 
-        if (TryComp<GasTankComponent>(component.GasTankEntity, out var gasTank) && gasTank.Air.Volume < Atmospherics.BreathVolume)
+        if (TryComp<GasTankComponent>(component.GasTankEntity, out var gasTank)
+            && gasTank.OutputPressure * Atmospherics.BreathVolume > gasTank.Air.TotalMoles * Atmospherics.R * gasTank.Air.Temperature)
```

Comment:
this seems like a weird check? cant it just check if the output pressure is >= the real pressure in the tank

Reply:
> this seems like a weird check? cant it just check if the output pressure is >= the real pressure in the tank

Thing is the pressure depends on the volume and temperature.
What we are checking here is if there is more moles in tank than we need.

This check seems weird indeed, so I just changed the operator position. This now check if (OutputPressure * BreathVolume / R * Temperature) (which equals to the number moles we need to breath) is superior or equal to gasTankTotalMoles.